chart/kubeapps/values.yaml

```diff
@@ -1,3 +1,9 @@
+## Global chart image registry
+## Please, note that this registry would be used for all the containers in the chart and dependencies
```

Comment:
I would reword this a bit:

> Please, note that this will override the image registry for all the images, including dependencies, configured to use the global value.

Reply:
`registry` might be too vague. It might make sense to call it dockerImageRegistry, imageRegistry or something like that.

> note that this registry would be used for all the containers in the chart and dependencies

I'd rephrase the comment by explaining what it does, the current message just says that it will affect to all the containers but not how.